scripts/codemod/helpers/codemodHelpers.js

```diff
@@ -0,0 +1,284 @@
+// @flow strict
+
+/**
+ * IMPORTANT NOTE
+ * Flow is used in these codemods with the only intention of providing better documentation on hover for each helper function while developing codemods and to increase code readability. Due to the complexity of Flow typing, we use generic types to prevent Flow form complaining. Simple props such as strings or booleans are typed to facilitate the usage of each helper function.
+ */
+
+// $FlowFixMe[unclear-type]
+type GenericType = any;
+
+type InitializeType = {
```

Comment:
Lots of open types in this file…can any/all of these be closed?

Reply:
DONE